### PR TITLE
Remove z2.installProducts. Use testing best practices.

### PR DESCRIPTION
**Pull request by tisto**
I think we can safely expect that we are not dealing with old zope2 only packages in 2015.

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -28,7 +28,7 @@ input = inline:
    python bootstrap-buildout.py --setuptools-version=8.3
    ${buildout:directory}/test.plone_addon/bin/buildout
    ${buildout:directory}/test.plone_addon/bin/test
-   rm -rf ${buildout:directory}/test.plone_addon/
+   # rm -rf ${buildout:directory}/test.plone_addon/
```
Comment (do3cc) [inline buildout.cfg]: Was this an accident?

Comment (tisto) [inline buildout.cfg]: Yeah. Kind of. For debugging I kept the generated code. Should not be committed though.

**Comment by cedricmessiant** [comment]
Sadly, we still have to deal with such packages in 2015. Two examples : Products.membrane and collective.taskqueue.pasplugin...

**Comment by djowett** [comment]
Um, will this affect uninstall profiles? I was trying to add a PR for uninstall profile & tests, but my uninstall method isn't being called. Is Zope2 required for that? (I'm looking at https://plone.org/products/collective-flowplayer/issues/5)

**Comment by djowett** [comment]
Ok, my question still stands, but it's  ab2eb80 rather than this change that is killing uninstall profiles

**Comment by tisto** [comment]
Sure. There might be cases where this is needed. Though, this does not justify to have that in all packages. We should just try to cover the most important use cases and try to keep the code as clean as possible. Otherwise we will end up with hundreds of packages that carry over this code lines without any good reason.

**Pull request merged**